pusher/src/Controller/IoSocketChatController.ts

```diff
@@ -217,14 +238,45 @@ export class IoSocketChatController {
                             userData.jabberId = jid(userIdentifier, EJABBERD_DOMAIN).toString();
                             if (EJABBERD_JWT_SECRET) {
                                 userData.jabberPassword = Jwt.sign({ jid: userData.jabberId }, EJABBERD_JWT_SECRET, {
-                                    expiresIn: "1d",
+                                    expiresIn: "30d",
                                     algorithm: "HS256",
                                 });
                             } else {
                                 userData.jabberPassword = "no_password_set";
                             }
                         }
 
+                        if (userData.userRoomToken && ADMIN_API_URL) {
+                            const jwtDecoded = jwtTokenManager.verifyJWTToken(
+                                userData.userRoomToken
+                            ) as unknown as UserRoomToken;
```

Comment:
You should use Zod for the `UserRoomToken` (rather than "as unknown as UserRoomToken" which is unsafe)